clarity_ext/domain/analyte.py

```diff
@@ -67,3 +68,12 @@ def updated_rest_resource(self, original_rest_resource, updated_fields):
             _updated_rest_resource.name = self.assigner.register_assign('name', self.name)
         return _updated_rest_resource, self.assigner.consume()
 
+    @property
+    def sample(self):
+        """
+        Returns a single sample for convenience. Throws an error if there isn't exactly one sample.
+
+        NOTE: There can be more than one sample on an Analyte. That's the case with pools.
+        """
+        return utils.single(self.samples)
```

Comment:
Aha. I guess that makes sense. I didn't realize the domain expert is expected to also write `analyte.samples[x]` if appropriate. Thanks. :+1: